app/src/end2endTest/java/gov/va/vro/end2end/util/TestSetup.java

```diff
@@ -12,26 +13,25 @@
 import java.time.format.DateTimeFormatter;
 import java.util.AbstractMap;
 import java.util.Map;
-import java.util.concurrent.atomic.AtomicInteger;
 
 /**
  * This class drives end-to-end tests based on two gold files in a resource directory. The file
  * assessment.json is the expected response from the server for assessment. The file
  * veteranInfo.json provides the veteran information needed for pdf generation.
  */
 @Getter
+@RequiredArgsConstructor
 public class TestSetup {
   private static final Map<String, String> diagnosticCodeToName =
       Map.ofEntries(
           new AbstractMap.SimpleEntry<>("7101", "Hypertension"),
           new AbstractMap.SimpleEntry<>("6602", "Asthma"));
-  private static final AtomicInteger claimSubmissionCounter = new AtomicInteger(7000);
+
+  private final String name;
 
   private String assessment;
   private String veteranInfo;
 
-  private String claimSubmissionId;
-
```

Comment:
Using `claimSubmissionId` based on `claimSubmissionCounter` is fragile and dependent on the order in which each test runs. For test assertions, `claimSubmissionId` is expected to be equal to the one in the json files.